packages/os/configure-boot-mount.service

```
@@ -0,0 +1,16 @@
+[Unit]
+Description=Configure mount points for the active boot partition
+ConditionPathExists=!/proc/vmcore
+RefuseManualStart=true
+RefuseManualStop=true
+Wants=prepare-local.service
+After=prepare-local.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/prairiedog configure-boot-mount
+RemainAfterExit=true
+StandardError=journal+console
+
+[Install]
+WantedBy=local-fs.target
```

Seems more consistent to give this the same treatment as other mount units:
```suggestion
[Install]
WantedBy=preconfigured.target
```